ingestors/tildelt-veileder-ingestor/src/main/kotlin/no/nav/amt/tiltak/ingestors/tildelt_veileder_ingestor/TildeltVeilederIngestor.kt

```diff
@@ -0,0 +1,7 @@
+package no.nav.amt.tiltak.ingestors.tildelt_veileder_ingestor
+
+interface TildeltVeilederIngestor {
```

Comment:
Trenger man å lage en ingestor her? Kan ikke dette egentlig ligge under Deltaker-modulen?

Reply:
Er ikke tanken bak ingestors at de skal være entrypoints inn til applikasjonen på lik linje som f.eks controllers. Føler det er en grei avgrensning at ingestoren tar imot ting fra kafka, parser det og kaller på riktig ting videre i servicene til f.eks deltaker modulen

Reply:
Ja, tanken min er vel at siden denne kun jobber med ting inne i deltaker så kan den bo der, slik at det blir enklere å skille ut den modulen om den blir for stor. Men kan godt bli i ingestor også.